Description: relax dependency on anyhow
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-02-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,7 +9,7 @@
 edition = "2018"
 
 [dependencies]
-anyhow = "1.0.53"
+anyhow = "1.0.44"
 clap = ">= 3.0.14, < 3.1.0"
 fern = { version = ">= 0.5.0, < 0.7.0", features = ["colored"] }
 globset = "0.4.8"
